runtime/lua/vim/filetype.lua

```diff
@@ -904,6 +904,12 @@ local extension = {
     end
     return "dosbatch"
   end,
+  rul = function(path, bufnr)
+    if getline(bufnr, 1, 6):lower():find("installshield") then
+      return "ishd"
+    end
+    return "diva"
```

Comment:
`else return "diva" end`?

(Sorry for taking out all those nitpicks on you; this is just the first time since the initial PR that there's a concerted look at these, and now is the best time to establish best practices.)

Reply:
I don't see a need to use `else` in these cases, but I might add if it's established practice

Reply:
I don't think an `else` should be added here either. Is there a previous example?

Reply:
The `asp` and `xpm` extensions above, for example.

Either way is fine with me, but I would prefer if the pattern was consistent throughout this file.